challenges/2021_cloud_base_height/02_explore_transformed_data.ipynb

```diff
@@ -0,0 +1,1532 @@
+{
```

Comment:
<p>As before a good idea to specify the environment that yo have tested this notebook with.</p><p> </p><p>Also good notebook practice I find to aim for smaller cells, I find it easier to read and also debug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>